include/stateMachineFwk/smEnumTypes.h

```diff
@@ -232,6 +235,7 @@ enum Action_e
 
 static constexpr const char* Actions[] =
```

Comment:
We should think how we can make sure that enum and text strings are in sync..Can you think of something using macros or something else ?

Reply:
Created task for this. Will fix this in the next iteration.